modules/misc/config.py

```diff
@@ -122,6 +122,7 @@ class Config:
 
     """Contains settings for the ground station process."""
 
+    general: dict[str, str] = field(default_factory=dict)
```

Comment:
Instead of using a dict, I'd also prefer you use a dataclass. See my above comment though, it would be better to just have an integer field `telemetry_buffer_length`.

This avoids having syntax like `config.general["field_name"]`, which doesn't allow the editor to autocomplete field names and also hides the structure of the config file from the code more since a dictionary can have any string as a key.